diagnostic.md

```diff
@@ -40,31 +56,38 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
 ```sh
-  # < Your Response Here >
+The serializer indicates what fiels will be passed through to the front end.
 ```
 
 ```rb
 class ProfileSerializer < ActiveModel::Serializer
+attributes :id, :given_name, :surname, :email
 end
 ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
 ```sh
-  # < Your Response Here >
+I do not believe you would  be able to scaffold a many to many relationship table. 
 ```
```

Comment:
If you can dream it, you can scaffold it! Probably.

`bundle exec rails g scaffold favorites movie:references profile:references 
`